src/Calculator/Controls/OverflowTextBlock.cpp

```diff
@@ -197,6 +197,7 @@ void OverflowTextBlock::UpdateScrollButtons()
             m_expressionContainer->Padding = Thickness(left, 0, right, 0);
```

Comment:
Why exactly do we need to update the Padding/Margin here? Can't we just apply those to xaml directly?

Reply:
it needs to be code-behind because we need to unregister to ViewChanged event first (to prevent UI cycle) and we need to call UpdateLayout and Measure after these values are set

Reply:
Makes sense, but why do we need to change the margin/padding in code-behind in the first place?

Reply:
Discussed in person